pkg/search/postgres/common.go

```diff
@@ -576,9 +576,10 @@ func RunDeleteRequestForSchema(schema *walker.Schema, q *v1.Query, db *pgxpool.P
 	}
 
 	queryStr := query.AsSQL()
-	_, err = db.Exec(context.Background(), queryStr, query.Data...)
+	tag, err := db.Exec(context.Background(), queryStr, query.Data...)
 	if err != nil {
 		return err
 	}
+	tag.RowsAffected()
```

Comment:
Is it planned to use the number of rows touched by the query ?

Reply:
Good point, I thought it might be useful but finally I have not used it. I'll remove it